index.html

```diff
@@ -3096,6 +3096,12 @@ <h3 id="applying">
           consistent.
         </p>
         <p>
+          To allow the community can easily find extensions, please add your
+          extensions to the <a href=
+          "https://github.com/w3c/manifest/wiki/Extensions-Registry">Extensions
+          Registry</a>.
```

Comment:
is the idea of "standard" and "proprietary" extensions similar to that of what IETF sometimes calls "experimental" and "permanent", with a different level of scrutiny when it comes to documentation and stability?